tests/Unit/Shared/Application/OpenApi/Builder/RequestPatchBuilderTest.php

```
@@ -40,4 +40,36 @@ public function testBuild(): void
+        $this->assertInstanceOf(RequestBody::class, $requestBody);
+    }
+
+    public function testBuildOptionalCreatesOptionalRequestBody(): void
+    {
+        $contextBuilderMock = $this->createMock(ContextBuilder::class);
+        $params = [];
+
+        $contextBuilderMock->expects($this->once())
+            ->method('build')
+            ->with($params)
+            ->willReturn(new \ArrayObject([]));
```

PHPMD (2.15.0)

Missing class import via use statement (line '68', column '30'). (undefined)

(MissingImport)